client/src/screens/TestDeckScreen.tsx

```diff
@@ -37,8 +37,8 @@ const generateTestDeckBallots = ({
   precinctId,
 }: GenerateTestDeckParams) => {
   const precincts: string[] = precinctId
-    ? [precinctId]
-    : election.precincts.map(p => p.id)
+			    ? [precinctId]
+			    : election.precincts.map(p => p.id)
```

Comment:
Battling formatters?

Reply:
I think we don't have the right hooks, but let me check.

Reply:
I think the issue is that Brian and I rely upon VS Code linting. Ben has been relying upon pre-commit hooks which exist in other repos but not yet in `election-manager`.